### PR TITLE
Cant navigate to library

### LINKED ISSUE #10600: Can't navigate to the library until I import some resources
## Observed behavior
[Reported by @MisRob and replicated by @radinamatic in the alpha 13 bug bash ](https://www.notion.so/learningequality/I-can-t-navigate-to-library-until-I-import-some-resources-3ac82561cb794a3fa91a6b0e79514171?pvs=4)

## Errors and logs

error1

## Expected behavior
Should be resolve at least partially in connection with https://github.com/learningequality/kolibri/issues/10599 which will allow users who are on a local network to browse other kolibris. 

We might need some clarification of expected behavior in other scenarios @jtamiace @rtibbles 

## User-facing consequences
User is unable to navigate 

## Context
v0.16.0-a13

### PR DESCRIPTION
**Pull request by akolson**
## Summary

This pr allows users to navigate to the library under the learn plugin even without importing some resources.
**Before:**
error1

**After:**

https://github.com/learningequality/kolibri/assets/5203639/c4db5e5b-b0c9-4ac2-8979-dc8a7fdc0e4a

## References

Fixes https://github.com/learningequality/kolibri/issues/10600

## Reviewer guidance

- Remember to run Zerotier. This should already be set up from earlier sessions, but you may need to reopen the application. Incase you need a refresher on setup, please visit [ZeroTier](https://www.notion.so/ZeroTier-3e508bf09b7e4c72a1713b8374a43cac)
- To view the Kolibri Library when exploring libraries, set the environment variable below before you run the app file
  - `KOLIBRI_CENTRAL_CONTENT_BASE_URL=https://unstable.studio.learningequality.org`
-  Setup a fresh kolibri installation. Do not import any channels.
- Navigate to `/learn/#/library` as illustrated in the attached(after) video.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by pcenov** [comment]
Hi @akolson this is implemented correctly as specified above.

Signed-in user:


Signed-in user who is able to see the Kolibri Library:



Signed-out user:



It seems that it would be nice to further discuss with @jtamiace what other usability improvements could be made here in that specific scenarios when there are no imported channels. Perhaps for a signed-in user under the channels tab there should be shown the link 'Import channels to your device'? And if the user is not signed in there should be also some explanatory text as just showing the Channels label is a bit confusing.

File: kolibri/plugins/learn/assets/src/routes/index.js [hunk 1]
```diff
@@ -98,10 +98,6 @@ export default [
     name: PageNames.LIBRARY,
     path: '/library' + optionalDeviceIdPathSegment,
     handler: to => {
-      if (!get(channels) || !get(channels).length) {
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/routes/index.js:101]: I think we may actually want to keep this, but only in the case that the user is not signed in. As guest users cannot browse other libraries/devices, it seems worthwhile to keep the `CONTENT_UNAVAILABLE` redirect specifically for them.

**Review by rtibbles — DISMISSED** [review]

File: kolibri/plugins/learn/assets/src/routes/index.js (hunk 1 above)
Comment (akolson) [reply kolibri/plugins/learn/assets/src/routes/index.js:101]: This has been resolved!

**Comment by jtamiace** [comment]
[This section of the Figma](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?type=design&node-id=1769%3A18049&t=bnXJABcy73WUGkAa-1) covers some of the different scenarios where there is still no content on the device, but there are other devices in the network that have content.  

The changes are:
* Changing the header text `Channels` to `Your library` and bumping that to the same size as `Other libraries`
* Adding the string `There is nothing in your library yet. Explore libraries around you and start adding materials to your own.` under the new header
* Removing the filter panel on the left since we are not allowing for filtering and search across channels in the network. I believe for performance reasons and technical complexity, the filters and keyword search would only apply to channels that are currently on the device.

Screenshot 2023-05-15 at 1 45 22 PM

**Comment by akolson** [comment]
Hello reviewer, feedback from design has been implemented and PR is now ready for a re-review.

**Review by rtibbles — COMMENTED** [review]
Code changes all make sense, and test coverage has been extended for these cases. A QA rereview to confirm conformance to the spec from @jtamiace and this is good to go!

**Comment by pcenov** [comment]
Hi @akolson this is looking great now! Comparing this final implementation with the Figma design I was wondering could you make just the following additional minor adjustments:
1. There should be less space between 'Your library' and the top navigation
2. There should be less space between 'Your library' and the text 'There is nothing in your library..."

**Comment by akolson** [comment]
Thanks @pcenov. Changes coming up shortly

**Comment by akolson** [comment]
Hi @pcenov. I have made the changes as requested. Thanks!
The result:
image

**Comment by pcenov** [comment]
Thanks @akolson the spacing is corrected now. At the same time while regression testing everything I was able to identify one more issue which I can also report separately if you wish - while refreshing the Library page when there is content I can briefly see the empty state of the page:

https://github.com/learningequality/kolibri/assets/79847249/793170d3-d444-48d5-99d9-49d1a13f8f71

**Comment by akolson** [comment]
Hi @pcenov, I think we should be able to resolve this as well with this pr. I currently manual testing the fix and should push shortly. Thanks

**Comment by akolson** [comment]
We should be good now @pcenov.

https://github.com/learningequality/kolibri/assets/5203639/45b549f3-d41a-4608-b24d-9f1afdf8f1de

**Comment by pcenov** [comment]
@akolson and @radinamatic - yes, LGTM as well, we should be good to go now. :)

**Review by radinamatic — APPROVED** [review]
Teamwork gets the job done! 👏🏽 🎉 :shipit:

**Review by rtibbles — APPROVED** [review]
No additional code questions from updates, let's merge!

**Pull request merged**